release-notes/release-5.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: April 17, 2024
+---
+
+# Features
+
+- [Oracle Database Integration]() with default dashboard
+- [Apache Networking Integration]() with default dashboard
+- Automated log collection with the Vercel APM for __Build__, __Lambda__, __Static__, __Edge__, and __External__ immediately upon installation
+- Ability to parse attributes from JSON log messages
+- Can use email addresses as query parameters within RUM to easily share user-specific sessions with your teammates
+- Users can set alerts based on their usage and budget on daily or monthly intervals
+- Advanced settings added to alerts. Users can configure alerts that will trigger when a certain specified threshold X is surpassed by Y number of occurrences over a given time interval Z
+- Global sidebar filter in the APM Page allows users to search across all of their instrumented application data
+- Dashboards can be dynamically generated based on a single metric that is being collected by Middleware
+- Filter the Span List within the APM Dashboard by parent `SpanID` to quickly identify what your'e looking for
+- Added __Critical__ severity level message that can be auto-detected across all logs
+
+# Changes
+
+- The Free Tier will be reset every month for all customers across all products (including paying users); this feature is live for new and existing users
+- Removed unnecessary graphs from the RUM Dashboard
+- Updated deletion confirmation message in the projects section to ensure users understand exactly what data they are removing from the Middleware platform
+- Dashboard Builder has a fully revamped dashboard and widget creation workflow that increases the speed by which you create new dashboards and widgets
+- More efficient flame graph processing in the APM dashboard to decrease loading time
+- Updated Kubernetes agent to reduce unnecessary logs, now less than the average DataDog agent
+- When users change their page from History mode to Live mode, the default time range displays data from five minutes in the past
+- Global Sidebar displays the filter value count for each selected item to better remember what it is your'e filtering
+- Added screen pagination on the Synthetic Monitoring page
+- Changed the Database Dashboard within the APM Page to have database requests, query times, and query operations as the new graphs on the landing page
+
+# Fixes
+
+- Fixed Slack Alert notification for failed synthetic monitors
```

Comment:
"Fixed an instance where Slack alerts were sending incorrectly for synthetic monitors"